index.html

```diff
@@ -1,140 +1,170 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-		<meta charset="utf-8">
-		<meta http-equiv="X-UA-Compatible" content="IE=edge">
-		<title>Git for Windows</title>
-		<meta name="description" content="We bring the awesome Git VCS to Windows">
-		<meta name="viewport" content="width=device-width, initial-scale=1">
-		<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700">
-		<link rel="stylesheet" href="css/pack.css">
-		<link rel="shortcut icon" href="favicon.ico">
-		<!--[if lt IE 9]>
+
+  <head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Git for Windows</title>
+	<meta name="description" content="We bring the awesome Git VCS to Windows">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<meta name="robots" content="all,follow">
+	<!-- Bootstrap CSS-->
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.5.3/css/bootstrap.min.css">
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,700">
+	<!-- Custom Stylesheet -->
+	<link rel="shortcut icon" href="favicon.ico">
+	<link rel="stylesheet" href="css/custom.css">
+	<!--[if lt IE 9]>
 		<script src="js/html5shiv.js"></script>
 		<script src="js/respond.js"></script>
 		<![endif]-->
-	</head>
-	<body>
-		<header>
-			<nav class="content">
-				<a href="index.html"><h1 class="gittext lowercase">Git<span>for Windows</span></h1></a>
-				<div class="version"><a href="https://github.com/git-for-windows/git/releases/tag/v2.34.0.windows.1" title="Version 2.34.0 was published on Mon, 15 Nov 2021 19:52:38 UTC">Version 2.34.0</a></div>
-				<ul class="list-unstyled">
-					<li><a style="font-weight:bold" href="https://github.com/git-for-windows/git/wiki/FAQ" target="_blank">FAQ</a></li>
-					<li><a href="https://github.com/git-for-windows/git" target="_blank">Repository</a></li>
-					<li><a href="https://groups.google.com/group/git-for-windows" target="_blank">Mailing List</a></li>
-				</ul>
-			</nav>
-			<section class="content feature">
-				<div class="imagerollover">
-					<img src="img/gwindows_logo.png" alt="Git Logo">
-					<img class="top" src="img/git_logo.png" alt="Git Logo">
+  </head>
+
+  <body>
+	<!-- navbar -->
+	<nav class="navbar navbar-light navbar-expand-lg fixed-top shadow-sm bg-white">
+		<a href="index.html" class="navbar-brand">
+			<h1 class="gittext lowercase">
+				Git<span>for Windows</span>
+			</h1>
```

Comment:
Having "Git for Windows" stylised as "git for windows" feels weird. It should probably be capitalised normally IMO.

Reply:
I don't mind the styling (it's carried over from the current incarnation), but I do mind the new font. Especially the `g` in `git`:

Current:
<img width="226" alt="current" src="https://user-images.githubusercontent.com/714283/142428988-23be55b2-e07e-4d63-b0ea-be74186d7334.png">

Proposed:
<img width="217" alt="proposed" src="https://user-images.githubusercontent.com/714283/142428951-0cb09db1-3ca8-47c8-861c-8ea412832aed.png">

The closed descender on the `g` felt more like a logotype.

Reply:
@mjcheetham I tried to keep true to the original design so the re-use.
@vtbassmatt, makes sense. Looks like I have to remain true to original font too.